model/schema/allele.yaml

```diff
@@ -831,15 +854,51 @@ slots:
     range: AlleleNoteSlotAnnotationDTO
     multivalued: true
 
-  allele_synonyms:
-    description: Synonyms of a given allele
+  allele_symbol:
+    description: >-
+      The one current accepted symbol for the allele: e.g., wg<sup>1</sup>.
+    exact_mappings:
+      - biolink:symbol
+    domain: Allele
+    range: AlleleSymbolSlotAnnotation
+    required: true
+    multivalued: false
+
+  allele_symbol_dto:
+    description: >-
+      The one current accepted symbol for the allele: e.g., wg<sup>1</sup>.
+    domain: AlleleDTO
+    range: SymbolSlotAnnotationDTO
+    multivalued: false
+    required: true
+    inlined: true
+
+  allele_full_name:
+    description: >-
+      The one current full name for an allele: e.g., wg<sup>1</sup>.
     domain: Allele
-    range: Synonym
+    range: AlleleFullNameSlotAnnotation
+    required: false
+    multivalued: false
+
+  allele_full_name_dto:
+    description: >-
+      The one current full name for an allele: e.g., wg<sup>1</sup>.
+    domain: AlleleDTO
+    range: FullNameSlotAnnotationDTO
+    multivalued: false
+    required: true
```

Comment:
I believe this shouldn't be required, only the allele_symbol_dto

Reply:
Good spot - I've removed the requirement